README.md

```diff
@@ -191,15 +191,15 @@ We have unit tests for the API/admin (Python) and for the React-based frontend
 
 For Python unit tests, run:
 ```sh
-docker-compose run --rm py.test
 docker-compose run --rm flake8  # linting
-docker-compose run --rm ptw # run tests whenever a file changes
+docker-compose run --rm py.test # run tests once
+docker-compose run --rm ptw # watch command to run tests whenever a file changes
 ```
 
 For JS unit tests, run:
 ```sh
-docker-compose run --rm npm test
-docker-compose run --rm npm run test:watch # run tests whenever a file changes
+docker-compose run --rm npm test # run tests once
+docker-compose run --rm npm run test:watch # watch command to run tests whenever a file changes
 docker-compose run --rm webpack       # lints (and builds)
```

Comment:
I think we can alleviate my concern by moving the `webpack` line above the `npm` commands -- then the final command will hang, but there's nothing after it, so that's okay.

Does that make sense? We've got non-devs running these commands, so I'm trying to think through potential pitfalls for them.